backend/core/scheduler.py

```diff
@@ -28,7 +27,7 @@ class Worker():
 
         def __init__(self, connection):
             self.connection = connection
-            self.time_to_wait = 1  # Time in secs to gather entries to perform a bulk operation
+            self.time_to_wait = float(os.getenv('BULK_TIMER', 1))  # Time in secs to gather entries to perform a bulk operation
```

Comment:
this is only for testing. we should not give access to the user to change the `BULK_TIME` so I won't be adding it to the wiki

Reply:
yes I agree